lib/model.js

```diff
@@ -842,7 +842,19 @@ class Model {
       }
     });
 
-    this.options.indexes = this.options.indexes.map(this._conformIndex);
+    this.options.indexes = this.options.indexes.map(index => {
+      index = this._conformIndex(index);
+      //Add unique indexes to rawAttributes
+      if (index.unique && index.fields) {
+        index.fields.forEach(field => {
```

Comment:
What will happen in case of unique index on two fields (composite unique index), will it mark both fields as `unique`? I would like to see a test which works in case of composite unique index